docs/release_notes/v1.12.3.md

```diff
@@ -5,6 +5,7 @@ This update includes bug fixes:
 - [Fix timeouts in HTTP service invocation when resiliency policies with timeouts are applied](#fix-timeouts-in-http-service-invocation-when-resiliency-policies-with-timeouts-are-applied)
 - [Fix dissemination of placement tables](#fix-dissemination-of-placement-tables)
 - [Fix SQL Server state store not working with correctly case-sensitive collations](#fix-sql-server-state-store-not-working-correctly-with-case-sensitive-collations)
+- [Fix Kafka Pub/sub consumer bottlenecks and long startup times when subscribing to multiple topics](#fix-kafka-pub/sub-consumers)
```

Comment:
Link was broken
```suggestion
- [Fix Kafka Pub/sub consumer bottlenecks and long startup times when subscribing to multiple topics](#fix-kafka-pubsub-consumer-bottlenecks-with-multiple-topics)
```

Reply:
Yeah forgot to save additional changes on local file before pushing